docs/_navbar.md

```diff
@@ -1,2 +1,2 @@
 * [Github](https://github.com/Martodox/animated-graph-generator)
-* [Youtube channel](https://www.youtube.com/playlist?list=PLPvM6BKGmMHePGnWuspLkqHhlb1F_9IJo)
```

Comment:
Not every video contains the graphs, change to 
https://www.youtube.com/watch?v=kodRKkPA-LU&list=PLPvM6BKGmMHfiwW9m8vXPvLQUmuErDn2f